instance/models/database_server.py

```diff
@@ -198,13 +213,112 @@ def default_port(self):
         return MYSQL_SERVER_DEFAULT_PORT
 
 
+class MongoDBReplicaSetManager(models.Manager):
+    """
+    Custom manager for the DatabaseServer model.
+    """
+    def _get_setting(self, field):
+        """
+        Get setting using prefix
+        """
+        return getattr(
+            settings,
+            self.model.DEFAULT_SETTINGS_NAME + '_' + field,
+            None
+        )
+
+    def get_replica_set_settings(self):
+        """
+        Create dictionary with replica settings
+        """
+        return {
+            'user': self._get_setting('USER'),
+            'password': self._get_setting('PASSWORD'),
+            'name': self._get_setting('NAME'),
+            'primary': self._get_setting('PRIMARY'),
+            'hosts': self._get_setting('HOSTS'),
+            'port': self._get_setting('PORT')
+        }
+
+    def _create_default(self):
+        """
+        Create the default database replica set configured in the Django settings, if any.
+        """
+        replica_settings = self.get_replica_set_settings()
+        for setting in replica_settings:
```

Comment:
@rocioar suggestion: this code gets the job done, but a bit unelegant. Do you think something like this ould work better:

```
optiona_fields = set("port")
required_fields = set(replica_settings.keys()) - optional_fields
all_required_present = all(replica_settings[field] is not None for field in required_fields)

if not all_required_present:
    logger.error(...)

Reply:
@e-kolpakov to be honest I rather use for and ifs than using all(), I think it's much more easier to read. But I will make a small change to see if looks better for you.